AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -631,18 +632,13 @@ extern NSInteger const ASDefaultDrawingPriority;
 @interface ASDisplayNode (UIViewBridge)
 
 /**
- * Marks the view as needing display. Convenience for use whether the view / layer is loaded or not. Safe to call from a background thread.
+ * Marks the view as needing display. Convenience for use whether the view / layer is loaded or not. Safe to call
+ * from a background thread.
  */
 - (void)setNeedsDisplay;
 
 /**
  * Marks the node as needing layout. Convenience for use whether the view / layer is loaded or not. Safe to call from a background thread.
- * 
- * If this node was measured, calling this method triggers an internal relayout: the calculated layout is invalidated,
- * and the supernode is notified or (if this node is the root one) a full measurement pass is executed using the old constrained size.
- *
- * Note: ASCellNode has special behavior in that calling this method will automatically notify 
```

Comment:
We should probably describe here (or file a documentation task) that if the node determines its own desired layout size will change, it will propagate the layout up the tree so its parents can have a chance to consider + apply the new size onto the node.

We could mention at that time that ASCellNode is capable of notifying the Table / Collection to adjust its size if such a layout adjustment occurs to it, just as the upward parent propagation does.